src/plugins/navigation_embeddable/public/components/navigation_embeddable_strings.ts

```diff
@@ -47,9 +47,34 @@ export const NavEmbeddableStrings = {
         i18n.translate('navigationEmbeddable.panelEditor.editFlyoutTitle', {
           defaultMessage: 'Edit links panel',
         }),
-      getSaveButtonLabel: () =>
-        i18n.translate('navigationEmbeddable.panelEditor.saveButtonLabel', {
-          defaultMessage: 'Save',
+      getAddToDashboardButtonLabel: () =>
+        i18n.translate('navigationEmbeddable.panelEditor.addToDashboardButtonLabel', {
+          defaultMessage: 'Add to dashboard',
+        }),
+      getAddToDashboardButtonTooltip: () =>
+        i18n.translate('navigationEmbeddable.panelEditor.addToDashboardButtonTooltip', {
+          defaultMessage: 'Add this links panel directly to this dashboard.',
+        }),
+      getSaveToLibraryButtonLabel: () =>
+        i18n.translate('navigationEmbeddable.panelEditor.saveToLibraryButtonLabel', {
+          defaultMessage: 'Save to library',
+        }),
+      getSaveToLibraryButtonTooltip: () =>
+        i18n.translate('navigationEmbeddable.panelEditor.saveToLibraryButtonTooltip', {
+          defaultMessage:
+            'Save this links panel to the library so you can easily add it to other dashboards.',
+        }),
+      getUpdateLibraryItemButtonLabel: () =>
+        i18n.translate('navigationEmbeddable.panelEditor.updateLibraryItemButtonLabel', {
+          defaultMessage: 'Update library item',
+        }),
+      getUpdateLibraryItemButtonTooltip: () =>
+        i18n.translate('navigationEmbeddable.panelEditor.updateLibraryItemButtonLabel', {
+          defaultMessage: 'Editing this panel might affect other dashboards.',
```

Comment:
Not sure if the "Update library item" button needs this tooltip, but I used similar copy to the `LibraryNotificationPopover`.

Reply:
I think it's nice for consistency that both buttons have tooltips! :+1: